### PR TITLE
Integrating dotnet/core with dotnet Jenkins CI

### PR DESCRIPTION
**Pull request by bryanar**
Placing netci.groovy to go along with the webhooks.

### CODE REVIEW

**Comment by bryanar** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        //   3. Adds standard parameters for PR and push jobs.
+        //      These allow PR jobs to be used for simple private testing, for instance.
+        // See the documentation for this function to see additional optional parameters.
+        Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
+        
+        // The following two calls add triggers for push and PR jobs
+        // In Github, the PR trigger will appear as "Windows Debug" and "Windows Release" and will be run
+        // by default
+        if (isPR) {
+            Utilities.addGithubPRTriggerForBranch(newJob, branch, "Windows ${configuration}")
```
Comment (mmitche) [inline netci.groovy]: Ubuntu?

Comment (bryanar) [inline netci.groovy]: yis. Thank you :)

**Comment by bryanar** [comment]
test ci please

**Comment by bryanar** [comment]
test this please

**Pull request merged**